scripts/edaby.coffee

```diff
@@ -0,0 +1,117 @@
+###
+  Description:
+    Ordering some food from eda.by to the office
+
+  Configuration:
+    CLIENT_NAME
+    CLIENT_EMAIL
+    CLIENT_PHONE
+    CLIENT_ADDRESS
+
+  Commands:
+    mona закажи нам <data> - it makes a request with prepared params
+    mona удали из заказа <data> - it will delete an item in the order
+    mona отправь заказ - it sends order to the EDA.BY
+
+  Author:
+    Dzmitry Chyrta
+###
+
+URL = 'http://eda.by/api.php'
+CLIENT_NAME = 'Х'
+CLIENT_EMAIL = 'Х'
+CLIENT_PHONE = '+375+(ХХ)+ХХХ-ХХ-ХХ'
+CLIENT_ADDRESS = 'Х'
```

Comment:
Let's use environment variables instead of constants. So it would be easier to configure it on server.